backend/src/feedback/feedback.gql

```diff
@@ -1,6 +1,11 @@
 input GenerateLink {
   email: String!
-  title: String
+    @stringLength(min: 0, max: 200)
+    @pattern(
+      regexp: "^(([^<>()[\\]\\\\.,;:\\s@\"]+(\\.[^<>()[\\]\\\\.,;:\\s@\"]+)*)|(\".+\"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$"
```

Comment:
nitpick:
Would there be a way to put this regex somewhere it could be imported both in frontend and backend?

Reply:
Good point 👍 though this would need a proper directive for the email. I'll create another task specifically for this.